React/my-app/src/App.js

```diff
@@ -1,19 +1,78 @@
-import React, { Component } from 'react';
-import logo from './logo.svg';
-import data from './data.json'
-import './App.css';
+import React, { Component } from "react";
+import data from "./data.json";
+import "./App.css";
+import defaultImg from "./photos/default.png";
 
 class App extends Component {
```

Comment:
create a new folder and call it `home`
and inside create the `smart` and the `dumb` file.
so
```
app
      src
           screens
                    home
                            index.js
                             layout.js
                            style.css
```
then, just import the home index to the application index